### ISSUE TITLE
Try plotting only logits most negatively affected, during ablations.

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#60]
The chat.py script seemed to suggest this. It seems like _rather more_ linearity than I had expected, to have logits be only meaningfully _downweighted_ during ablations. But let's see, at scale.
